doc/examples/applications/plot_image_comparison.py

```diff
@@ -81,7 +85,8 @@
 #
 # `blend` is the result of the average of the two images.
 
-fig = plt.figure(figsize=(8, 9))
+
```

Comment:
```suggestion
```

Reply:
Reverted. plt.plot() is now replaced with plt.show()